### PR TITLE
Add error page for HTTP 400

### PR DESCRIPTION
**Pull request by alexluckett**
Supports 368034 to present a friendly error rather than JSON if the user changes the URL parameters.

### CODE REVIEW

File: designer/server/src/views/400.njk [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% extends 'layouts/page.njk' %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
```
Comment (colinrotherham) [inline designer/server/src/views/400.njk:2]: Slowly trying to switch to double quotes everywhere in Nunjucks

```suggestion
{% extends "layouts/page.njk" %}

{% from "govuk/components/button/macro.njk" import govukButton %}
```

File: designer/server/src/views/400.njk [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+{% block content %}
+  {% call appPageBody({
+    heading: pageHeading
+  }) %}
+    <p class="govuk-body">The service couldn't process your request.</p>
```
Comment (colinrotherham) [inline designer/server/src/views/400.njk:8]: What does "process your request" mean to a user?

Watch out for the [negative contraction](https://www.gov.uk/guidance/style-guide/a-to-z#contractions) "couldn't" too

```suggestion
    <p class="govuk-body">The service could not process your request.</p>
```

File: designer/server/src/views/400.njk (hunk 1 above)
Comment (alexluckett) [reply designer/server/src/views/400.njk]: Done for a few pages I found

**Review by davidjamesstone — APPROVED** [review]

File: designer/server/src/views/400.njk (hunk 2 above)
Comment (alexluckett) [reply designer/server/src/views/400.njk]: I've put the question to Dan, will look to merge this unless they weigh in and let them improve it later.

**Pull request merged**